app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -182,42 +179,62 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
     @Override
     public void onResume() {
         super.onResume();
-        checkReloadDisk(false);
-        int cursor = _hlEditor.getSelectionStart();
-        cursor = Math.max(0, cursor);
-        cursor = Math.min(_hlEditor.length(), cursor);
-        _hlEditor.setSelection(cursor);
+        onPrepareOptionsMenu(getFragmentMenu());
+    }
+
+    @Override
+    public void onStart() {
+        super.onStart();
 
         AppSettings appSettings = new AppSettings(getContext());
         _hlEditor.setGravity(appSettings.isEditorStartEditingInCenter() ? Gravity.CENTER : Gravity.NO_GRAVITY);
-        if (_document != null && _document.getFile() != null) {
-            if (!_document.getFile().getParentFile().exists()) {
-                //noinspection ResultOfMethodCallIgnored
-                _document.getFile().getParentFile().mkdirs();
-            }
-            boolean permok = _shareUtil.canWriteFile(_document.getFile(), false);
-            if (!permok && !_document.getFile().isDirectory() && _shareUtil.canWriteFile(_document.getFile(), _document.getFile().isDirectory())) {
-                permok = true;
-            }
-            if (_shareUtil.isUnderStorageAccessFolder(_document.getFile()) && _shareUtil.getStorageAccessFrameworkTreeUri() == null) {
+
+        try {
+            File file = _document.getFile();
+            File parent = file.getParentFile();
+            if (parent.exists()) parent.mkdirs();
+
+            boolean permok = _shareUtil.canWriteFile(file, false);
+            _textSdWarning.setVisibility(permok ? View.GONE : View.VISIBLE);
+
+            if (_shareUtil.isUnderStorageAccessFolder(file) && _shareUtil.getStorageAccessFrameworkTreeUri() == null) {
                 _shareUtil.showMountSdDialog(getActivity());
                 return;
             }
-            _textSdWarning.setVisibility(permok ? View.GONE : View.VISIBLE);
+
+            if (getActivity() instanceof DocumentActivity) {
+                if (file.getAbsolutePath().contains("mordor/1-epub-experiment.md")) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [These nested if statements could be combined](https://app.codacy.com/manual/gsantner/markor/pullRequest?prid=5359905)